pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
We would need to vendor the object file then (`tcptracer-ebpf.o`) if we want to use go-generate on scope.